src/main/webapp/WEB-INF/pages/userform.jsp

```diff
@@ -567,14 +561,14 @@
 <script type="text/template" id="apiKeyDetailsTemplate">
 	{{#enabled}}
     <div class="api-menu__key col-xs-8">
-      {{#key}}
-        <strong>Key</strong>: <span id="api-menu__keyValue">{{key}}</span>
+      {{#revokable}}
+        <strong>Key</strong>: <span id="api-menu__keyValue"> ... </span>
```

Comment:
should we display some text here instead like the message from the api 'API key details cannot be accessed when 'operating' as another user'? or maybe disable the button and add a tooltip explaining?

Reply:
ok yes, I've changed "operating as" error to be a standard error, which UI can report the normal way (error popup).